lib/widgets/autocomplete.dart

```diff
@@ -212,8 +250,16 @@ class ComposeAutocomplete extends AutocompleteField<MentionAutocompleteQuery, Me
           children: [
             avatar,
             const SizedBox(width: 8),
-            Text(label),
-          ])));
+            Column(
+              mainAxisSize: MainAxisSize.min,
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+              Text(label, style: TextStyle(fontSize: 18,  height: 1.1, color: designVariables.contextMenuItemLabel) //Creates a line height equavalent to ~20
+                  .merge(weightVariableTextStyle(context, wght: 600))),
+              Visibility(
```

Comment:
Instead of `Visibility`, how about using Dart's ["collection if" feature](https://dart.dev/language/collections#control-flow-operators):

```dart
if (metaData.isNotEmpty) Text(/* … */),
```

, but combined with my earlier suggestions it would be

```dart
if (metadata != null) Text(/* … */),
```